src/main/java/org/gridsuite/filter/server/utils/FiltersUtils.java

```diff
@@ -107,4 +131,496 @@ public static boolean isRangeNominalVoltage(Double nominalV, Double minValue, Do
     public static boolean isEnergySource(Generator generator, String energySource) {
         return generator.getEnergySource().name().equalsIgnoreCase(energySource);
     }
+
```

Comment:
Maybe add these methods to another file to avoid polluting this one? FilterServiceUtils?

Reply:
Done (only 2 public methods called in FilterService put on new FilterServiceUtils class)